docs/noencap-hybrid-modes.md

```diff
@@ -0,0 +1,174 @@
+
+* Run a routing protocol or even manually configure routers to add routes to
+the Node network routers. For example, Antrea can work with [kube-router](https://www.kube-router.io)
+and leverage kube-router to advertise Pod CIDRs to routers using BGP. Section
+[Using kube-router for BGP](#using-kube-router-for-bgp) describes how to
+configure Antrea and kube-router to work together.
+
+When the Node network can support forwarding and routing of Pod traffic, Antrea
+can be configured to run in the `NoEncap` mode, by setting the `trafficEncapMode`
+config option of `antrea-agent` to `noEncap`. By default, Antrea performs SNAT
+(source network address translation) for the outbound connections from a Pod to
+outside of the Pod network, using the Node's IP address as the SNAT IP. In the
+`NoEncap` mode, as the Node network knows about Pod IP addresses, the SNAT by
+Antrea might be unnecessary. In this case, you can disable it by setting the
+`noSNAT` config option to `true`. The `trafficEncapMode` and `noSNAT` config
+options are defined in `antrea-agent.conf` of the `antrea` ConfigMap in the
+[Antrea deployment YAML](https://github.com/vmware-tanzu/antrea/blob/main/build/yamls/antrea.yml).
+
+```yaml
+  antrea-agent.conf: |
+    ... ...
+    # Determines how traffic is encapsulated. It has the following options:
+    # encap(default):    Inter-node Pod traffic is always encapsulated and Pod to external network
+    #                    traffic is SNAT'd.
+    # noEncap:           Inter-node Pod traffic is not encapsulated; Pod to external network traffic is
+    #                    SNAT'd if noSNAT is not set to true. Underlying network must be capable of
+    #                    supporting Pod traffic across IP subnets.
+    # hybrid:            noEncap if source and destination Nodes are on the same subnet, otherwise encap.
+    # networkPolicyOnly: Antrea enforces NetworkPolicy only, and utilizes CNI chaining and delegates Pod
+    #                    IPAM and connectivity to the primary CNI.
+    #
+    trafficEncapMode: noEncap
+
+    # Whether or not to SNAT (using the Node IP) the egress traffic from a Pod to the external network.
+    # This option is for the noEncap traffic mode only, and the default value is false. In the noEncap
+    # mode, if the cluster's Pod CIDR is reachable from the external network, then the Pod traffic to
+    # the external network needs not be SNAT'd. In the networkPolicyOnly mode, antrea-agent never
+    # performs SNAT and this option will be ignored; for other modes it must be set to false.
+    #noSNAT: false
+    ... ...
+```
+
+After changing the options, you can deploy Antrea with the `noEncap` mode by
+applying the deployment YAML.
+
+### Using kube-router for BGP
+
+We can run kube-router in the advertisement-only mode to advertise Pod CIDRs to
+the peered routers, so the routers can know how to route Pod traffic to the
+Nodes. To deploy kube-router in the advertisement-only, first download the
```

Comment:
To deploy kube-router in advertisement-only mode

Reply:
Done.